### PR TITLE
Define and activate release 11.6.0 for AWS

### PR DESCRIPTION
**Pull request by pipo02mix**

### CODE REVIEW

File: aws/v11.6.0/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# :zap: Giant Swarm Release v11.6.0 for AWS :zap:
+
+This release updates Cert-Manager `1.1.0`.
```
Comment (cokiengchiara) [inline aws/v11.6.0/README.md:3]:
```suggestion
This release updates Cert-Manager to `1.1.0` to add support for Route53 using Kiam annotation.
```

Propose to add the main value of the release to the main paragraph.

**Review by cokiengchiara — APPROVED** [review]
LGTM (release notes part at least)

File: aws/v11.6.0/release.diff [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp        giantswarm.io/docs: https://docs.giantswarm.io/reference/cp
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v11.5.6                                                 |    name: v11.6.0
+spec:                                                              spec:
+  apps:                                                              apps:
+  - name: cert-exporter                                              - name: cert-exporter
+    version: 1.2.3                                                     version: 1.2.3
+  - componentVersion: 0.9.0                                          - componentVersion: 0.9.0
+    name: cert-manager                                                 name: cert-manager
+    version: 1.0.8                                                     version: 1.1.0
```
Comment (alex-dabija) [inline aws/v11.6.0/release.diff:14]: Why isn't this line marked as being different when it's clear that the version numbers are different?

File: aws/v11.6.0/release.diff [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    name: node-exporter                                                name: node-exporter
+    version: 1.2.0                                                     version: 1.2.0
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    version: 1.0.0                                                     version: 1.0.0
+  - name: aws-cni                                                    - name: aws-cni
+    version: 1.7.3                                                     version: 1.7.3
+  - name: aws-operator                                               - name: aws-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 8.7.9                                              |      version: 8.7.9
```
Comment (alex-dabija) [inline aws/v11.6.0/release.diff:47]: Why is this line marked as changed when the version numbers are the same?

**Comment by pipo02mix** [comment]
@alex-dabija done PTAL

**Review by alex-dabija — APPROVED** [review]

**Pull request merged**